### ISSUE TITLE
Add Styled & Responsive password.liquid view

### ISSUE THREAD

**Issue by ScottJr** [Shopify/Timber#480]
This would be a welcome addition for launching stores to allow users to preview pages without having to style for mobile.  Lots of clients I send my themes to in development are confused by the non-responsive Shopify default.

**Comment by carolineschnapp** [comment]
https://docs.shopify.com/manual/configuration/store-customization/page-specific/password-page/can-i-customize-my-password-page

**Comment by ScottJr** [comment]
@carolineschnapp Yeah I mean that is obvious.  But just like the rest of the framework it comes responsive out of the box.  There is no responsive element to that page.
